service/task.go

```diff
@@ -581,6 +581,7 @@ func (uis *UIServer) taskLogRaw(w http.ResponseWriter, r *http.Request) {
 	}
 
 	raw := (r.FormValue("text") == "true") || (r.Header.Get("Content-Type") == "text/plain")
+	printPriority := r.FormValue("priority") == "true" || !raw
```

Comment:
Does printing the priority for non-raw logs also affect other logs like HTML, or is it just Parsley? If it does affect other logs and the other loggers won't render the priority specially, is there a way to make prioriting printing a Parsley-specific thing?

Reply:
Nvm, after some Slack discussion, I didn't realize there's already [logic to strip away the priority for HTML logs](https://github.com/evergreen-ci/evergreen/blob/f55f276ee90e3b2ee65330bb360aa15c5b807c4e/apimodels/task_log.go#L160-L170).